easybuild/easyconfigs/c/CLHEP/CLHEP-2.1.3.1-intel-2015a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'CMakeMake'
+
+name = 'CLHEP'
+version = '2.1.3.1'
+
+homepage = 'http://proj-clhep.web.cern.ch/proj-clhep/'
+description = """The CLHEP project is intended to be a set of HEP-specific foundation and
+ utility classes such as random generators, physics vectors, geometry and linear algebra.
+ CLHEP is structured in a set of packages independent of any external package."""
+
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCELOWER_TGZ]
+source_urls = ['http://proj-clhep.web.cern.ch/proj-clhep/DISTRIBUTION/tarFiles/']
+
+dependencies = [ ('CMake', '3.1.3') ]
+
+# build dir should not be in the source tree!
+preconfigopts = 'mkdir ../../2.1.3.1_bld && cd ../../2.1.3.1_bld &&'
+prebuildopts = 'cd ../../2.1.3.1_bld &&'
+preinstallopts = 'cd ../../2.1.3.1_bld &&'
+
+configopts = '-DCMAKE_BUILD_TYPE=Release' 
+
+# build dir should not be in the source tree!
+srcdir = '../2.1.3.1/CLHEP/'
```

Comment:
don't hardcode the version please

how does this relate to the `preconfigopts` & co?